prepare_tetragon_artifacts.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+
+git_checkout_ref() {
+  local dir="$1"
+  local ref="$2"
+  git --work-tree="$dir" --git-dir="$dir/.git" checkout -q -f "$ref"
+  git --work-tree="$dir" --git-dir="$dir/.git" clean -xdfq
+}
+
+if [ $# -lt 1 ] ; then
+	echo "usage: $0 <TETRAGON_TAG>" 1>&2
+	exit 1
+fi
+
+TAG=$1
+VERSION="${TAG:1}"
+if [ -f "tetragon-${VERSION}.tgz" ]; then
+  echo "tetragon-${VERSION}.tgz already exists. Nothing to do."
+  exit 0
+fi
+
+echo "Generating tetragon package from tag: $TAG"
+rm -rf tetragon
+git clone git@github.com:cilium/tetragon.git
```

Comment:
Oh, I didn't see this. Can we avoid cloning tetragon every time? Does this work correctly the 2nd and 3rd time?

Reply:
Though actually if we move this towards a github action then maybe we don't even need to run it locally so we'll need the clone after all...

Reply:
oh and now I see the `rm -rf tetragon`, that solves the clone problem. Bit slow but I guess it works.

Reply:
yeah once we move it to github actions we can just use the checkout action 👍